catalog/core/catalog-core-standardframework/src/main/java/ddf/catalog/cache/impl/MetacardComparator.java

```diff
@@ -0,0 +1,70 @@
+/**
+ * Copyright (c) Codice Foundation
+ * <p>
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser
+ * General Public License as published by the Free Software Foundation, either version 3 of the
+ * License, or any later version.
+ * <p>
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without
+ * even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details. A copy of the GNU Lesser General Public License
+ * is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ */
+package ddf.catalog.cache.impl;
+
+import com.google.api.client.repackaged.com.google.common.base.Strings;
+
+import ddf.catalog.data.Metacard;
+
+public class MetacardComparator {
+
+    private MetacardComparator() {
+    }
+
+    public static boolean isSame(Metacard cachedMetacard, Metacard latestMetacard) {
```

Comment:
@lessarderic and @Lambeaux i broke it apart because checkstyle doesn't like have 8 comparisons, I can see how i can make it easier to read, but i think i can check the checksum first since that's the first attribute i check anyway